### ISSUE TITLE
Compatibility issue with cy-api library leads to "Domain exception: log stack null."

### ISSUE THREAD

**Issue by FFdhorkin** [archfz/cypress-terminal-report#122]
When we upgraded cypress-terminal-report from 3.1.2 to the more recent 3.3.x line, we started having what initially looked like a regression on https://github.com/archfz/cypress-terminal-report/issues/98

![image](https://user-images.githubusercontent.com/59899746/137563751-863d24da-bb3c-4f15-8deb-fde3a916e67a.png)

However, it looks like the new issue is something different. It seems to only happen in conjunction with the cy-api library. ( https://github.com/bahmutov/cy-api ). I don't believe it's an issue with cy-api itself, as using cypress-terminal-report 3.1.2 makes this issue go away.

Here's a new minimal repro suite:
```
describe('Error repro', () => {
    it('Test1', () => {
        return cy.api({
            url: 'http://google.com',
            method: 'GET',
        });
    });

    it.skip('test2', () => {
    });
});
```

It appears that as long as you have a test that makes a cy.api call and THEN skip another test, you get the error. If you remove the skipped test, or replace cy.api with cy.request, we don't get the error

**Comment by FFdhorkin** [comment]
I doubt it will be of much use, but here's the full stack trace when I expand the screenshotted error from above:
[stacktrace.txt](https://github.com/archfz/cypress-terminal-report/files/7356379/stacktrace.txt)

**Comment by archfz** [comment]
I have found the issue. But it is not caused by the `cy.api`, it was indeed a core issue with the extended controller + skipping tests. Now I cannot write a test to prevent regression on this, cause in fact for me on latest cypress 8.6.0 it doesn't throw the error and break the tests, it only shows it in console.

**Comment by archfz** [comment]
Fix released in 3.3.4
